packages/react-router/lib/components.tsx

```diff
@@ -220,49 +223,28 @@ export function Outlet(props: OutletProps): React.ReactElement | null {
   return useOutlet(props.context);
 }
 
-interface DataRouteProps {
```

Comment:
Got rid of the duplication here - `<Route>` props are now just `RouteObject` equivalents that expect `ReactNode` children instead of `ReactElement` children